docs/toc.yml

```diff
@@ -1156,12 +1153,19 @@
 
 - name: Patterns and practices
   items:
-  - name: Share code from VSTO Add-in to Office add-in
-    href: tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md
   - name: Batching custom function remote calls
     href: excel/custom-functions-batching.md
     displayName: Excel, Custom Functions
   - name: Authentication
     href: develop/authentication-pnp.md
   - name: Handle high DPI and DPI scaling in your Office solution
     href: /office/client-developer/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution
+
+- name: VSTO transition resources
+  items: 
+    - name: VSTO add-in developers' guide
+      href: overview/learning-path-transition.md
+    - name: Share code from VSTO Add-in to Office add-in
+      href: tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md
+    - name: Identify an equivalent COM add-in
```

Comment:
Did this "Identify an equivalent COM add-in" get moved from somewhere in the TOC?

Reply:
Yep, came from some other node that dealt with generic concepts or something

Reply:
So should there be a deleted entry for it in the TOC file?